CMakeLists.txt

```diff
@@ -39,7 +41,37 @@ catkin_package(
   INCLUDE_DIRS include
 )
 
+#######################
+# Include directories #
+#######################
+
+include_directories(
+  tests
+  include
+  ${EIGEN3_INCLUDE_DIR}
+  ${GTEST_INCLUDE_DIRS}
+  ${catkin_INCLUDE_DIRS}
+  ${${yaml_cpp_name}_INCLUDE_DIRS}
+)
+
+#########
+# tests #
+#########
+
+set(GTEST_FILES ${GTEST_FILES}
+  tests/GtestMain.cpp
```

Comment:
Sorry to be annoying, but the convention for the file naming is "lower_case_with.underscore"

Reply:
All good !! I will adhere to naming conventions